assets/dictionary.txt

```diff
@@ -205,3 +205,4 @@ bytecode
 eval
 asyncStore
 CPG
+debbuger.html
```

Comment:
Ohh, it also have two `b`s :P

Reply:
Yes, I added the misspelled version("debbuger.html") to `dictionary.txt`. As I understood it from `docs/pull-requests.md`, the dictionary.txt should have the misspelled words. Is that correct?

This doesn't matter as much as I've removed it, but would be useful to clarify for reference later.